deploy/charts/trust-manager/values.yaml

```diff
@@ -182,6 +182,13 @@ app:
   # The verbosity of trust-manager logging. This takes a value from 1-5, with the higher value being more verbose.
   logLevel: 1
 
+  # Leader Election Configuration
+  leaderElection:
+    # How long will a lease be valid? You probably want to increase this to 60-120s.
+    leaseDuration: 15s
+    # How long will a controller try to renew a lease before stepping down? You probably want to increase this to 30-45s.
```

Comment:
```suggestion
    # The interval between attempts by the acting leader to renew a leadership slot before it stops leading.
    # This MUST be less than or equal to the lease duration.
    # The default should be sufficient in a healthy cluster but can be slightly increased to prevent trust-manager from restart-looping when the API server is overloaded.
```